src/pages/shoppingListsPage/shoppingListsPage.test.tsx

```diff
@@ -208,4 +229,183 @@ describe('<ShoppingListsPage />', () => {
       })
     })
   })
+
+  describe('creating a shopping list', () => {
+    describe('when successful', () => {
+      const mockServer = setupServer(
+        getGamesAllSuccess,
+        getShoppingLists,
+        postShoppingLists
+      )
+
+      beforeAll(() => mockServer.listen())
+      beforeEach(() => mockServer.resetHandlers())
+      afterAll(() => mockServer.close())
+
+      describe('when there is an existing aggregate list', () => {
```

Comment:
In a ways having both this and the next `describe` block tests the MSW mocks more than the actual app - I just want to make sure that it is setting `shoppingLists` to the values returned from the API, not just adding the created list to the view.